routes/auth.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const router = express.Router();
+const {
+  login,
+  logout,
+  loginCallback,
+  loginSuccess,
+  loginFailure,
+} = require("../controllers/auth");
+
+router.route("/").get(login);
```

Comment:
```suggestion
authRouter
  .get("/", login)
  .get("/callback", loginCallback)
  .get("/logout", logout)
  .get("/success", loginSuccess)
  .get("/fail", loginFailure);
```
이렇게 refactoring 해주면 더 가독성이 좋아질 것 같아요